gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
I don't think this is needed. We are caching things from '/dist' (that's what rootdir points to), which is after the page has been assembled, so it doesn't really matter what it was assembled from. My understanding is that dynamicURlToDependencies is needed when you do client side templating, which we don't